app/views/idv/question.html.slim

```diff
@@ -0,0 +1,35 @@
+- title t('upaya.titles.idv.p2')
+
+.clearfix.mxn2
+  .sm-col-7.px2.mx-auto
+    .panel
+      .panel-heading
+        h2.m0 = 'Verify identity'
+      .mb1.h6.caps Question 1
+      =form_tag({ controller: 'idv', action: 'answer' }, method: 'post')
+        =hidden_field_tag('question_key', quest.key)
+        h3.mt2.mb2
+          =quest.display
+        -quest.choices.each do |choice|
+          label.radio.block.mb1
+            = radio_button_tag('answer',
+                    choice.key, false, id: "choice_#{ choice.key }")
```

Comment:
i think we should add `required: true` to the radio_button_tag so the user can't submit the form unless they select something